### DISCUSSION TITLE
Feasy Beacons may not be compatible

### DISCUSSION THREAD

**Discussion by MrGibbage** [ESPresense/ESPresense#424] [Support]
I got two of these beacons yesterday:
https://www.amazon.com/Distance-Vibration-Consumption-Waterproof-Bluetooth/dp/B07P95JWJ3
The problem I am having is that no matter what I set the transmit power to, I can only detect the beacon if it is within a meter of the ESP32. I am using MQTT explorer so I can watch the messages coming in, and when I move the beacon away even two meters, the messages stop. Same thing with both beacons.
I already have some BlueCharm beacons and they work fine, so it's not a problem with my setup.
Not sure if there is anything else I can try, so I am pretty sure I'll just return these. It's a shame because I needed some waterproof beacons and those are the only ones I can find on amazon. Unless someone has some ideas....

**Comment by DTTerastar** [comment]
Set your max distance to 0 for testing (that disables the filtering based on distance).  Check your rss@1m.  That value is most likely the issue.  I'd bet it's claiming a value that is way higher than seen which will make all the distance values super high.
